docs/troubleshooting/README.md

```diff
@@ -1 +1,611 @@
+    uid: 37e83327-26e0-4c48-8276-e62cc6aa067f
+  resourceVersion: "10085"
+  uid: f2a3d0ea-c9fa-455d-be09-51b5d090e5d6
+spec:
+  policy:
+    affinity:
+      clusterAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          clusterSelectorTerms:
+          - labelSelector:
+              matchLabels:
+                env: prod
+    placementType: PickN
+  policyHash: ZjE0Yjk4YjYyMTVjY2U3NzQ1MTZkNWRhZjRiNjQ1NzQ4NjllNTUyMzZkODBkYzkyYmRkMGU3OTI3MWEwOTkyNQ==
+status:
+  conditions:
+  - lastTransitionTime: "2023-11-27T21:33:01Z"
+    message: could not find all the clusters needed as specified by the scheduling
+      policy
+    observedGeneration: 1
+    reason: SchedulingPolicyUnfulfilled
+    status: "False"
+    type: Scheduled
+  observedCRPGeneration: 2
+  targetClusters:
+  - clusterName: kind-cluster-1
+    clusterScore:
+      affinityScore: 0
+      priorityScore: 0
+    reason: picked by scheduling policy
+    selected: true
+  - clusterName: kind-cluster-2
+    reason: 'ClusterUnschedulable, none of the nonempty required cluster affinity
+      term (total number: 1) is matched'
+    selected: false
+```
+
+The solution here is to add the **env:prod** label to the member cluster resource for **kind-cluster-2** as well so that the scheduler can pick the cluster to propagate resources.
+
+### How to verify the latest ClusterSchedulingPolicySnapshot for a CRP?
+
+- We need to have ClusterResourcePlacement's name **{CRPName}**, replace **{CRPName}** in the command below,
+
+```
+kubectl get clusterschedulingpolicysnapshot -l kubernetes-fleet.io/is-latest-snapshot=true,kubernetes-fleet.io/parent-CRP={CRPName}
+```
+
+- Compare ClusterSchedulingPolicySnapshot with the **CRP's** policy to ensure they match (excluding numberOfClusters field from **CRP's** spec)
+- The ClusterSchedulingPolicySnapshot has a label called **number-of-clusters** check to see if it matches the number of clusters requested in **CRP's** **PickN** placement policy.
+
+### How can I debug when my CRP status is ClusterResourcePlacementSynchronized condition status is set to "False"?
+
+**ClusterResourcePlacementSynchronized** condition status is set to **"False"** if the following occurs, the work is not created/updated for a new **ClusterResourceSnapshot**, **ClusterResourceBinding** for a given cluster.
+
+In the **ClusterResourcePlacement** status section check to see which **placementStatuses** also has WorkSynchronized status set to **false**.
+
+From the **placementStatus** we can get the **clusterName** and then check the **fleet-member-{clusterName}** namespace to see if a work objects exists/updated in this case it won't as **WorkSynchronized** has failed.
+
+We need to find the corresponding **ClusterResourceBinding** for our **ClusterResourcePlacement** which should have the status of **work** create/update.
+
+A common case where this could happen is user input for the **rollingUpdate** config it too strict for rolling update strategy.
```

Comment:
```suggestion
A common case where this could happen is user input for the **rollingUpdate** config it too strict for rolling update strategy.

Please check the **rollingUpdate** strategy to make sure the maxUnavailable and maxSurge meets your expectations.
```